BREAKING_CHANGES.md

```diff
@@ -1 +1,7 @@
-# Breaking Changes
\ No newline at end of file
+# Breaking Changes
+
+## 2.0.0
+
+* Added support .net 7
+* Dropped support for .net 6
+* Refactored IBiometryService KeyName to Key and KeyValue to Value
```

Comment:
Say that you renamed the method parameters.

Reply:
* Renamed method parameters of `IBiometryService` from `keyName` to `key` and from `keyValue` to `value`.